python/cuml/cuml_accel_tests/upstream/summarize-results.py

```diff
@@ -331,6 +328,12 @@ def format_table(rows, col_sep="  "):
     return formatted_rows
 
 
+def get_test_id(testcase) -> str:
```

Comment:
Changes in this file are just to always include the fully qualified test id in output from the `summarize-results.py` script.